src/pages/settings/PaymentsPage.js

```diff
@@ -1,24 +1,110 @@
 import React from 'react';
-import HeaderWithCloseButton from '../../components/HeaderWithCloseButton';
-import Navigation from '../../libs/Navigation/Navigation';
+import {Pressable, TextInput, View} from 'react-native';
+import PropTypes from 'prop-types';
+import {withOnyx} from 'react-native-onyx';
+
```

Comment:
Why the blank line ?

Reply:
@Dal-Papa It's just for separating imports in groups like imports from the library, from projects files. I can remove that. 

I would suggest having lint rules for ordered/sorted imports. What do you think? For now, I will remove this blank line.

Thanks for the review 🙌